internal/pkg/mount/switchroot/switchroot.go

```diff
@@ -98,6 +100,12 @@ func Switch(prefix string, mountpoints mount.Points) (err error) {
 		log.Printf("race detection enabled with halt_on_error=1")
 	}
 
+	if val := procfs.ProcCmdline().Get("talos.fips140"); val != nil && fipsmode.Enabled() {
```

Comment:
no documentation here yet, this is mostly for testing only - we don't provide a FIPS build yet